src/riscv-integration.adoc

```diff
@@ -1013,6 +1013,8 @@ Unlike machine and supervisor level CSRs, {cheri_base_ext_name} does not require
 The <<pcc>> is made visible in a CSR. This provides access to an
 <<infinite-cap>> capability while in debug mode without executing <<AUIPCC>>.
 
+<<pcc>> resets to the <<infinite-cap>> with the address field set to the core boot address.
```

Comment:
Please note that the <<infinite-cap>> expands to the string `Infinity`, so this would sound better in the PDF if it was written like this:
```
<<pcc>> resets to the <<infinite-cap>> capability with the address ...
```